.travis.yml

```diff
@@ -24,9 +24,8 @@ cache:
 
 script:
   # need to override as the default is to test
-  - sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))'
   - sbt -J-XX:ReservedCodeCacheSize=256m +update
-  - if [[ "$TRAVIS_BRANCH" == "master" ]]; then sbt whitesourceCheckPolicies; fi
+  - if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_EVENT_TYPE" == "push" ]]; then sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))' whitesourceCheckPolicies whitesourceUpdate; else sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))' whitesourceCheckPolicies; fi 
```

Comment:
That's the way I interpret it, too: valid values for `TRAVIS_EVENT_TYPE` are "push, pull_request, api, cron" (https://docs.travis-ci.com/user/environment-variables/), so I assumed this would be 'push' when building after merging. We'll see for sure after merging :).